ts/components/ToastManager.stories.tsx

```diff
@@ -27,6 +27,14 @@ function getToast(toastType: ToastType): AnyToast {
       return { toastType: ToastType.AlreadyGroupMember };
     case ToastType.AlreadyRequestedToJoin:
       return { toastType: ToastType.AlreadyRequestedToJoin };
+    case ToastType.AttachmentDownloadStillInProgress:
+      // TODO I have no idea what parameters to return here
+      return {
+        toastType: ToastType.AttachmentDownloadStillInProgress,
+        parameters: {
+          count: 1,
+        },
```

Comment:
It's whatever permutation you want to see in storybook when you type `yarn dev` and navigate to the `ToastManager` story...

Reply:
Ahh thanks, I see. 
I haven't worked with this storybook thing yet, but in this case the parameters seem reasonable.